code/style.css

```diff
@@ -19,10 +24,104 @@ h1 {
   color: white;
 }
 
+img {
+  width: 100%;
+  height: auto;
+}
+
+.margin-bottom {
+  margin-bottom: 12px;
+ }
+
+.hidden {
+  display: none;
+}
+
+
+/******** Sparkle Animation ********/
```

Comment:
Kind of unrelated but wanted to share this cool starry night, built using only css: https://codepen.io/WebSonick/pen/nBPZZO 😁